From 9b59f153072a9eb330b67b125de9cc55a6a8ff74 Mon Sep 17 00:00:00 2001
From: Vinh Ngoc Khuc <vkhuc@apache.org>
Date: Tue, 5 Aug 2014 03:46:55 +0000
Subject: [PATCH] OPENNLP-704 Fixed a bug in SentenceDetectorTool where
 performance monitor is not started. Thanks to Eugen Hanussek for providing
 the fix.

git-svn-id: https://svn.apache.org/repos/asf/opennlp/trunk@1615859 13f79535-47bb-0310-9956-ffa450edef68
---
 .../opennlp/tools/cmdline/sentdetect/SentenceDetectorTool.java   | 1 +
 1 file changed, 1 insertion(+)

diff --git a/opennlp-tools/src/main/java/opennlp/tools/cmdline/sentdetect/SentenceDetectorTool.java b/opennlp-tools/src/main/java/opennlp/tools/cmdline/sentdetect/SentenceDetectorTool.java
index 802f374e1..3aaf27468 100644
--- a/opennlp-tools/src/main/java/opennlp/tools/cmdline/sentdetect/SentenceDetectorTool.java
+++ b/opennlp-tools/src/main/java/opennlp/tools/cmdline/sentdetect/SentenceDetectorTool.java
@@ -60,6 +60,7 @@ public void run(String[] args) {
       SentenceDetectorME sdetector = new SentenceDetectorME(model);
 
       PerformanceMonitor perfMon = new PerformanceMonitor(System.err, "sent");
+      perfMon.start();
 
       try {
         ObjectStream<String> paraStream = new ParagraphStream(new PlainTextByLineStream(new SystemInputStreamFactory(),
